bin/env.sh

```diff
@@ -20,10 +20,12 @@ export deviceVirtual=nexus3.edgexfoundry.org:10004/docker-device-virtual-go:mast
 export appService=nexus3.edgexfoundry.org:10004/docker-app-service-configurable:latest
 export systemManagement=nexus3.edgexfoundry.org:10004/docker-sys-mgmt-agent-go:master
 
-export vault=nexus3.edgexfoundry.org:10004/docker-edgex-secret-store-go:master
+export vault=vault:1.0.3
 export vaultWorker=nexus3.edgexfoundry.org:10004/docker-edgex-security-secretstore-setup-go:master
 export kongdb=postgres:9.6
 export kong=kong:1.3.0-centos
 export edgexProxy=nexus3.edgexfoundry.org:10004/docker-edgex-security-proxy-setup-go:master
+export securitySecretsSetup=nexus3.edgexfoundry.org:10004/docker-edgex-secrets-setup-go:master
+export securitySecretsSetup=edgexfoundry/docker-edgex-secrets-setup-go:master-dev
```

Comment:
Waiting for https://github.com/edgexfoundry/edgex-go/pull/2225.
Remove line 29 (master-dev) once required fixes are merged and overnight build runs.